src/TextField/TextField.jsx

```diff
@@ -370,6 +391,12 @@ const TextField = React.createClass({
     if (this.props.onKeyDown) this.props.onKeyDown(event);
   },
 
+  _handleInputKeyUp() {
+    this.setState({currentCharCount: this.getValue().length});
+    if (this.props.charCountMax >= this.getValue().length) this.setState({charCountError: false});
```

Comment:
Yes @newoga that will be a good idea but how do you suggest to manipulate the error style in TextCharCount?  Right now it is easy to manipulate error style of label and underline using state of currentCharCount as both are defined in TextField